abnf/odata-abnf-construction-rules.txt

```diff
@@ -909,12 +909,12 @@ primitiveLiteral = nullValue                  ; plain values up to int64Value
                  / geometryPoint
                  / geometryPolygon
 
-; in Atom and JSON message bodies and CSDL DefaultValue attributes
-primitiveValue = booleanValue
-               / guidValue
+; in CSDL XML DefaultValue attributes
+primitiveValue = boolean

```

Comment:
If string values are not mentioned here, better remove the rule completely, along with its test cases.